apps/emqx_ds_builtin_raft/src/emqx_ds_replication_layer.erl

```diff
@@ -1153,6 +1153,8 @@ update_iterator(ShardId, OldIter, DSKey) ->
             Err
     end.
 
+handle_custom_event(_DBShard, _Latest, ra_tick) ->
```

Comment:
This MODULE, has two  
-behaviour(emqx_ds_buffer).
-behaviour(ra_machine).

The change is to make `tick` from raft should not be handled by emqx_ds_storage_layer